### DISCUSSION TITLE
Additional contributor relation: Copyist

### DISCUSSION THREAD

**Discussion by halhosar** [Edirom/MerMEId#139] [Ideas]
Greeting, MerMEIds all! I am working on several cataloguing projects for eighteenth-century composers, and I was wondering if it might be possible to include the term "copyist" as a possible relation for contributors? This is a very well-established term for this period that is not entirely congruent with "scribe", the closest term at the moment.

**Comment by peterstadler** [comment]
Hi @halhosar,

the contributor relation within MerMEId are based on the established ["MARC Code List for Relators"](https://www.loc.gov/marc/relators/relaterm.html). Within this list a distinction is made between (at least) two types of "copyists": 

* Art copyist [acp]
* Music copyist [mcp]

So, while a simple copyist is not available from this list, I guess a "Music copyist" is what you are looking for (and is available from MerMEId's dropdown menu)?
